packages/liferay-npm-scripts/src/scripts/build.js

```diff
@@ -39,17 +39,19 @@ function compileBabel() {
  * `liferay-npm-bundler` executable.
  */
 function runBundler() {
-	moveToTemp(CWD, '.npmbundlerrc');
+	try {
+		moveToTemp(CWD, '.npmbundlerrc');
 
-	const RC_PATH = path.join(CWD, '.npmbundlerrc');
+		const RC_PATH = path.join(CWD, '.npmbundlerrc');
 
-	fs.writeFileSync(RC_PATH, JSON.stringify(BUNDLER_CONFIG));
+		fs.writeFileSync(RC_PATH, JSON.stringify(BUNDLER_CONFIG));
 
-	spawnSync('liferay-npm-bundler');
+		spawnSync('liferay-npm-bundler');
```

Comment:
If the command fails at this point, how does this resolve? Won't we have two `.npmbundlerrc` files? One from line 47, and the original being restored in line 53.

Reply:
Based on my testing, it puts everything back the way it was (with one caveat, noted below):

https://github.com/liferay/liferay-npm-tools/blob/d23683598ebde1132b2d88fc69c848dd5931ac4e/packages/liferay-npm-scripts/src/utils/move-to-temp.js#L15-L44

- Lines 25-26 of `move-to-temp.js` check for `$PWD/TEMP-.npmbundlerrc` and rename it back to `$PWD/.npmbundlerrc` (overwriting the config that we just wrote). If there is no such file (ie. if `moveToTemp()` didn't create one), nothing happens, which means that the config that was written out will stay there and show up in the `git status` output as an untracked file.
- Line 29 *would* check for config in `$PWD/package.json` and restore it, but does nothing here because we're not passing a `packageKey`.

So, a more complete solution would remove the created  `.npmbundlerrc` in the case that one didn't exist beforehand, but #71 was just asking that we not mangle an pre-existing file, so I didn't go that far.